content/14-passing-arguments-to-another-function-in-javascript/post.md

```diff
@@ -0,0 +1,27 @@
+Title: Passing arguments to another function in JavaScript
+
+-----
+
+Date: 1428527874
+
+-----
+
+Contributors: rasshofer
+
+-----
+
+Text:
+
+If you want to pass arguments to another function in JavaScript, you can use `.apply()` to have the same access to `arguments`.
+
+```js
+function a() {
+    b.apply(this, arguments);
+}
+
+function b() {
+    console.log(arguments);
+}
+
+a('Lorem', 'ipsum');​
+```
```

Comment:
Maybe add a note that `arguments` is an Array-like thing, but shouldn't be used as such without conversion. Is that out of scope?